csharp/Platform.Collections/Lists/ListFiller.cs

```diff
@@ -8,6 +8,12 @@ public class ListFiller<TElement, TReturnConstant>
         protected readonly List<TElement> _list;
         protected readonly TReturnConstant _returnConstant;
 
+        /// <summary>
+        /// <para>Pass the value of variables to their counterparts, for further safe operation, using access modifier protected and a readonly type.</para>
```

Comment:
Look at example here: https://github.com/linksplatform/Ranges/blob/b8736a67af3f02aed55464c541a3d93f4d808cbf/csharp/Platform.Ranges/Range%5BT%5D.cs#L45-L51 You can do comment here by analogy.